app/Http/Controllers/ControllerPositionController.php

```diff
@@ -14,4 +14,17 @@ public function getAllControllers() : JsonResponse
     {
         return response()->json(ControllerPosition::all());
     }
+
+    public function getControllerPositionsDependency() : JsonResponse
+    {
+        $positions = [];
+        ControllerPosition::all()->each(function (ControllerPosition $position) use (&$positions) {
```

Comment:
Could you use the Laravel collection `mapWithKeys` here? https://laravel.com/docs/master/collections#method-mapwithkeys

Reply:
Has been updated :)